--- haskell-language-server-1.7.0.0/haskell-language-server.cabal~	2022-05-02 22:15:49.004888017 +0800
+++ haskell-language-server-1.7.0.0/haskell-language-server.cabal	2022-05-02 22:21:40.237277996 +0800
@@ -39,7 +39,7 @@
     , extra
     , filepath
     , text
-    , prettyprinter
+    , prettyprinter >= 1.7
 
 -- Default warnings in HLS
 common warnings

Diff finished.  Mon May  2 22:21:48 2022
